pkg/compose/up.go

```diff
@@ -124,7 +146,7 @@ func (s *composeService) Up(ctx context.Context, project *types.Project, options
 		return err
 	})
 
-	if options.Start.Watch {
+	if options.Start.Watch && !options.Start.NavigationMenu {
```

Comment:
IIUC `--watch` would then **not** start watch mode until menu is disabled. Why ?

Reply:
If the --watch is passed and the menu is active, this will start the watch command here https://github.com/docker/compose/pull/11605/files#diff-809980e7b57a811e276a5d7951e759d3071d157ebad94f0a52370476a1fbfcedR103-R105

Reply:
right. yet more spaghetti code to our already confusing "architecture" 😅